src/recipes/jam-band.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "<Jam Band>",
```

Comment:
You can remove all the placeholder tokens in this file:
```suggestion
    "name": "Jam Band",
```

Reply:
I believe I followed requested changes, thank you for reviewing.